.github/workflows/preinstall-deps.yml

```diff
@@ -0,0 +1,33 @@
+name: Preinstall Dependencies
+
+on:
+  workflow_call:
+    inputs:
+      node-version:
+        required: false
+        default: '16.x'
+        type: string
+
+jobs:
+  build-sth:
+    name: Install Dependencies ${{ inputs.node-version }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Nodejs ${{ inputs.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ inputs.node-version }}
+
+      - name: Cache node_modules
+        uses: actions/cache@v3
+        with:
+          path: |
+            node_modules
+          key: ${{ runner.os }}-${{ inputs.node-version }}-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        run: yarn install --ignore-engines --frozen-lockfile
```

Comment:
I couldn't find any references to this workflow. What is it used for?

Reply:
I used it but it didn't help. I'd like to use it later for accelerating the dependency install, but for that I don't want to extend the complexity. Future code.